UnofficialCrusaderPatch/Localization/English.txt

```diff
@@ -2,6 +2,31 @@
 //         DEV
 // ======================
 
+ai_resources_rebuy
+{
+"Improve AI restocking of needed goods"
+}
+
+ai_resources_rebuy_descr
+{
+"The AI often doesn't restock goods even if it has the gold for it, leaving its production chains running dry. This option reduces the amount of time the AI waits for its stocks to be empty before restocking the necessary goods."
+}
+
+ai_nowood_maxtime_descr
+{
+"Time in 1/12th of months until the AI restocks 5 wood."
```

Comment:
If we can't have the long description, then maybe this short description should be a little more descriptive:
"The AI waits for empty stocks to refill by themselves before buying the goods in the market. Set the time in 1/12th of months until the AI buys 5 wood."